### ISSUE TITLE
cargo pgx test fails for want of a test schema

### ISSUE THREAD

**Issue by ethanpailes** [zombodb/pgx#42]
It seems that `cargo pgx test` fails to run normally. It looks like there is some wrapper SQL that is not getting generated properly.

Here is some info about my system:

```
$ uname -a
Linux closet 4.19.0-9-amd64 #1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07) x86_64 GNU/Linux
$ rustc --version
rustc 1.45.2 (d3fb005a3 2020-07-31)
```

Here is the full output of `cargo pgx test`. This is running against the default template created by `cargo pgx new`:

```
$ cargo pgx test
    Finished test [unoptimized + debuginfo] target(s) in 0.10s
     Running target/debug/deps/nql-faedc460d5f14e6c

running 1 test
building extension with features `pg10 pg_test`
"cargo" "build" "--features" "pg10 pg_test" "--no-default-features"
   Compiling nql v0.0.0 (/home/ethan/repos/nql)
    Finished dev [unoptimized + debuginfo] target(s) in 1.96s

installing extension
     Copying control file to `/home/ethan/.pgx/10.13/pgx-install/share/postgresql/extension/nql.control`
     Copying shared library to `/home/ethan/.pgx/10.13/pgx-install/lib/postgresql/nql.so`
     Writing extension schema to `/home/ethan/.pgx/10.13/pgx-install/share/postgresql/extension/nql--1.0.sql`
    Finished installing nql
"/home/ethan/.pgx/10.13/pgx-install/bin/postmaster" "-D" "/home/ethan/repos/nql/target/pgx-test-data-10" "-h" "localhost" "-p" "32210", pid=5186
PostgreSQL 10.13 on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
[2020-09-06 17:57:46.072 EDT] [5186] [5f555b59.1442]: LOG:  listening on IPv6 address "::1", port 32210
[2020-09-06 17:57:46.072 EDT] [5186] [5f555b59.1442]: LOG:  listening on IPv4 address "127.0.0.1", port 32210
[2020-09-06 17:57:46.183 EDT] [5186] [5f555b59.1442]: LOG:  listening on Unix socket "/tmp/.s.PGSQL.32210"
[2020-09-06 17:57:46.351 EDT] [5188] [5f555b5a.1444]: LOG:  database system was shut down at 2020-09-06 17:32:44 EDT
test tests::pg_test_hello_nql ... FAILED

failures:

---- tests::pg_test_hello_nql stdout ----
installing extension
    Creating database pgx_tests
thread 'tests::pg_test_hello_nql' panicked at '
[2020-09-06 17:57:46.072 EDT] [5186] [5f555b59.1442]: LOG:  listening on IPv6 address "::1", port 32210
[2020-09-06 17:57:46.072 EDT] [5186] [5f555b59.1442]: LOG:  listening on IPv4 address "127.0.0.1", port 32210
[2020-09-06 17:57:46.183 EDT] [5186] [5f555b59.1442]: LOG:  listening on Unix socket "/tmp/.s.PGSQL.32210"
[2020-09-06 17:57:46.396 EDT] [5186] [5f555b59.1442]: LOG:  database system is ready to accept connections
...
[2020-09-06 17:57:48.175 EDT] [5204] [5f555b5c.1454]: LOG:  statement: BEGIN
[2020-09-06 17:57:48.176 EDT] [5204] [5f555b5c.1454]: LOG:  statement: SELECT "tests"."test_hello_nql"();
[2020-09-06 17:57:48.176 EDT] [5204] [5f555b5c.1454]: ERROR:  schema "tests" does not exist at character 8
[2020-09-06 17:57:48.176 EDT] [5204] [5f555b5c.1454]: STATEMENT:  SELECT "tests"."test_hello_nql"();
[2020-09-06 17:57:48.176 EDT] [5204] [5f555b5c.1454]: LOG:  statement: ROLLBACK

schema "tests" does not exist
Postgres location: namespace.c:3023
Rust location: <unknown>

', /home/ethan/.cargo/registry/src/github.com-1ecc6299db9ec823/pgx-tests-0.0.13/src/framework.rs:120:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    tests::pg_test_hello_nql

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

Stopping Postgres

error: test failed, to rerun pass '--lib'
     [error] cargo pgx test failed with status = Some(101)
```

**Comment by eeeebbbbrrrr** [comment]
I'm pretty certain this is due to upstream changes in `proc-macro2`.  I've pushed a few commits to resolve this and will be publishing a new pgx release within the hour.

**Comment by eeeebbbbrrrr** [comment]
released in 0.0.14
